http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/AutoDeleteAppMonitor.scala

```diff
@@ -54,7 +53,7 @@ class AutoDeleteAppMonitor[F[_]](
       _ <- a.cloudContext match {
         case CloudContext.Gcp(googleProject) =>
           if (a.appStatus == AppStatus.Error) {
-            implicit val implicitAppContext = Ask.const(AppContext(traceId, now))
+            implicit val implicitAppContext: Ask[F, AppContext] = Ask.const(AppContext(traceId, now))
```

Comment:
IntelliJ suggested adding this type annotation.  I can remove it if you don't like it.

Reply:
it's good to have type annotated